### PR TITLE
Add end-to-end UI tests

### PR DESCRIPTION
**Pull request by prybalko**
Add end-to-end "happy path" UI test

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ module.exports = {
   extends: [
     "eslint:recommended",
     "plugin:vue/recommended",
-    "plugin:prettier/recommended",
```
Comment (prybalko) [inline .eslintrc.js:6]: Upgraded `prettier` v2 has some breaking changes comparing to v1. I didn't want to add visual noise with new formatting to this PR, so I disabled it for now. I'll update the formatting in the next PR

File: .github/workflows/yarn.yml [hunk 2]
```diff
@@ -7,7 +7,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2-beta
+
+      - name: Read .nvmrc
+        run: echo NVMRC=`cat .nvmrc` >> $GITHUB_ENV
```
Comment (prybalko) [inline .github/workflows/yarn.yml:12]: use the version from `.nvmrc`

File: .gitignore [hunk 3]
```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 *.sln
 *.sw*
 
-.test-results
+test-results
```
Comment (prybalko) [inline .gitignore:23]: this seems to be the default path

**Review by AndreiEres — APPROVED** [review]
lgtm

**Review by TarikGul — APPROVED** [review]
lgtm

**Review by kirushik — APPROVED** [review]
:shipit:

**Pull request merged**